src/toil/cwl/cwltoil.py

```diff
@@ -177,7 +177,7 @@ def run(self, fileStore):
 class CWLJob(Job):
     """Execute a CWL tool wrapper."""
 
-    def __init__(self, tool, cwljob):
+    def __init__(self, tool, cwljob, **kwargs):
```

Comment:
You probably want to name this argument `executor_options`, too. Or document this argument in the docstring.